test/integration/multiplexed_upstream_integration_test.cc

```diff
@@ -26,7 +26,8 @@ class MultiplexedUpstreamIntegrationTest : public HttpProtocolIntegrationTest {
 
   void bidirectionalStreaming(uint32_t bytes);
   void manySimultaneousRequests(uint32_t request_bytes, uint32_t max_response_bytes,
-                                uint32_t num_streams = 50);
+                                uint32_t num_streams = 50,
+                                std::chrono::milliseconds timeout = TestUtility::DefaultTimeout);
```

Comment:
IIUC the timeout is only used for debugging the disabled test. Let's remove it for now, as it only impacts the disabled test.

Reply:
done.